fhttp/httprunner.go

```diff
@@ -90,7 +90,13 @@ func RunHTTPTest(o *HTTPRunnerOptions) (*HTTPRunnerResults, error) {
 	if o.SequentialWarmup {
 		warmupMode = "sequential"
 	}
-	log.Infof("Starting http test for %s with %d threads at %.1f qps and %s warmup", o.URL, o.NumThreads, o.QPS, warmupMode)
+
+	connReuseMsg := fmt.Sprintf("with connection reuse range between %d and %d", o.ConnReuseRange[0], o.ConnReuseRange[1])
+	if o.ConnReuseRange == [2]int{0, 0} {
+		connReuseMsg = ""
```

Comment:
pedantic as this section isn't high performance but in general, avoid doing work to then throw away, so:
```suggestion
	connReuseMsg := ""
	if o.ConnReuseRange != [2]int{0, 0} {
		connReuseMsg = fmt.Sprintf(" with connection reuse [%d, %d]", o.ConnReuseRange[0], o.ConnReuseRange[1])
```
note I also added a space so we can remove the extra one below